velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -304,6 +304,8 @@ TEST_F(ParquetReaderTest, parseReadAsLowerCase) {
   const std::string upper(getExampleFilePath("upper.parquet"));
 
   dwio::common::ReaderOptions readerOptions{leafPool_.get()};
+  auto outputRowType = ROW({"a", "b"}, {BIGINT(), BIGINT()});
```

Comment:
Let's modify the `readBinaryAsStringFromNation` test instead.

Reply:
Thanks @majetideepak, updated the test `readBinaryAsStringFromNation `, and modified the column names in schema of `nation.parquet` to camelCase for testing this config. Could you please take another look?

Reply:
@pramodsatya Thanks. I did not think the parquet file would need to be modified. But it does. Can we revert to your previous test?

Reply:
Also, this test seems to segfault. Can you take a look?

Reply:
Got it, moved the test back, the segfault appears to be related to the test changes and it is fixed now.